include/psp2kern/net/net.h

```diff
@@ -706,10 +706,14 @@ int ksceNetSocket(const char *name, int domain, int type, int protocol);
 int ksceNetAccept(int s, SceNetSockaddr *addr, unsigned int *addrlen);
 int ksceNetBind(int s, const SceNetSockaddr *addr, unsigned int addrlen);
 int ksceNetListen(int s, int backlog);
-int ksceNetRecv(int s, void *buf, unsigned int len, int flags);
+int ksceNetRecv(int s, void *buf, unsigned int len, int flags); //!< This function does not actually exist in Kernel, so it's a #define to ksceNetRecvfrom() for compatibility
 int ksceNetRecvfrom(int s, void *buf, unsigned int len, int flags, SceNetSockaddr *from, unsigned int *fromlen);
-int ksceNetSend(int s, const void *msg, unsigned int len, int flags);
+int ksceNetSend(int s, const void *msg, unsigned int len, int flags); //!< This function does not actually exist in Kernel, so it's a #define to ksceNetSendto() for compatibility
```

Comment:
1. I removed it from `db.yml` because it was incorrect.
2. `#define` is the right way to go. I added the function prototype as well so that it appears in the documentation with the rest of the functions.